go/pkg/client/client.go

```diff
@@ -510,33 +510,42 @@ func (c *Client) WaitExecution(ctx context.Context, req *repb.WaitExecutionReque
 	return res, nil
 }
 
-// GetCapabilities wraps the underlying call with specific client options.
-func (c *Client) GetCapabilities(ctx context.Context, req *repb.GetCapabilitiesRequest) (res *repb.ServerCapabilities, err error) {
+// GetBackendCapabilities returns the capabilities for a specific server connection
+// (either the main connection or the CAS connection).
+func (c *Client) GetBackendCapabilities(ctx context.Context, conn *grpc.ClientConn, req *repb.GetCapabilitiesRequest) (res *repb.ServerCapabilities, err error) {
 	opts := c.RPCOpts()
 	err = c.Retrier.Do(ctx, func() (e error) {
 		return c.CallWithTimeout(ctx, func(ctx context.Context) (e error) {
-			res, e = c.capabilities.GetCapabilities(ctx, req, opts...)
+			res, e = regrpc.NewCapabilitiesClient(conn).GetCapabilities(ctx, req, opts...)
```

Comment:
Now we no longer need to save it as an instance variable.

Reply:
Ah yes, you mentioned before but I forgot to actually do it 🤦